### PR TITLE
Skipping reboot on Supervisor for Cisco 8000 distributed chassis

### PR DESCRIPTION
**Pull request by vperumal**
### Description of PR
Skipping reboot on Supervisor for Cisco 8000 distributed chassis. 

Summary:
Fixes # (issue)

### Type of change

- Fill x for your type of change.
- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

#### What is the motivation for this PR?
Currently we are hitting an intermittent issue if Supervisor is rebooted during a test. To avoid tests to get errored out during nightly run, we are skipping reboot for supervisor node

### CODE REVIEW

**Comment by SuvarnaMeenakshi** [comment]
Can we add the test cases to be skipped here: https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml.
We could open a github issue and add that under condition along with platform string of supervisor.

**Comment by vperumal** [comment]
> Can we add the test cases to be skipped here: https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml.
> We could open a github issue and add that under condition along with platform string of supervisor.

Hi @SuvarnaMeenakshi, the check for skip using test_mark_conditions happens at the start of the script even before dut is selected. It works fine in a single dut case but in case of multi-dut wherein it is randomly picked up within the testcase, test_mark_conditions does not work. Hence had to put it in each individual testcase.

**Comment by vperumal** [comment]
@abdosi , added skip for watchdog reboot here - since I closed - https://github.com/sonic-net/sonic-mgmt/pull/6812.

**Review by abdosi — CHANGES_REQUESTED** [review]
please add close pid platforms also.

File: tests/acl/test_acl.py [hunk 1]
```diff
@@ -1102,6 +1102,8 @@ def post_setup_hook(self, dut, localhost, populate_vlan_arp_entries, tbinfo):
 
         """
         dut.command("config save -y")
+        if dut.is_supervisor_node and dut.facts['platform'] in ['x86_64-8800_rp_o-r0']:
+            pytest.skip("Skip reboot for Supervisor card in Cisco 8000 distributed chassis")
```
Comment (abdosi) [inline tests/acl/test_acl.py:1106]: Remove this .  this should not get call for supervisor.

**Comment by abdosi** [comment]
we will not need now as reboot is working fine

**Pull request closed without merging**